ChangeLog.md

```diff
@@ -11,6 +11,33 @@ GitHub release pages and in the git history.
 
 ## \[Unreleased\]
 
+### Python API
+
+- Add `spglib.get_magnetic_spacegroup_type_from_symmetry` for getting a magnetic space-group type information from magnetic symmetry operations
+- `spglib.get_spacegroup_type` and `spglib.get_spacegroup_type_from_symmetry` now return a `spglib.SpaceGroupType` object, inheriting `dataclasses.dataclass`, instead of a plain `dict`. This object also has the `dict`-like interface, for example
+
+```python
+spgtype: SpaceGroupType | None = get_spacegroup_type(cell)
+assert spgtype is not None
+print(spgtype['number'])
+print(list(spgtype.items()))
+```
+
+However, we recommend using the `dataclass` interface:
```

Comment:
This would be better in a `### Main changes` (particularly if there are deprecation notices). In the `Python API`, just mention which methods were changed.